cmd/kpod/diff.go

```diff
@@ -74,25 +74,22 @@ func formatJSON(output []diffOutputParams) (diffJSONOutput, error) {
 }
 
 func diffCmd(c *cli.Context) error {
-	if len(c.Args()) != 1 {
-		return errors.Errorf("container, layer, or image name must be specified: kpod diff [options [...]] ID-NAME")
-	}
 	if err := validateFlags(c, diffFlags); err != nil {
 		return err
 	}
-	config, err := getConfig(c)
-	if err != nil {
-		return errors.Wrapf(err, "could not get config")
+
+	if len(c.Args()) != 1 {
+		return errors.Errorf("container, layer, or image name must be specified: kpod diff [options [...]] ID-NAME")
```

Comment:
nit of a nit.  Could you alphabetize this?  "container, image, or layer name....."

Reply:
I think we want all errors to be lower case - so I was told a while back. Unless it is only for errors that start out like "error getting ..."

Reply:
??? I'm just suggesting changing the ordering of the words in the message here, not the casing.  I'd just switch places for the word "layer" and "image".  A) it will put them in alphabetical order, B) it puts them in the order of likely use for this command IMO.

Reply:
oh whoops! I don't know why I read alphabetize as capitalize - my bad :/ Will fix.

Reply:
fixed.